cosmos/scripts/chaos/cosmosdb_client_network_chaos_tool/readme.md

```diff
@@ -0,0 +1,144 @@
+|delayInMs| The delay/latency induced in the network in milliseconds. Only required when performing Network Delay chaos.
+---
+
+- Note for Optional parameters:
+    - cosmosDBMasterKey and cosmosDBIdentityClientId cannot be null at the same time. At least one of them should be provided. If both cosmosDBMasterKey and cosmosDBIdentityClientId are provided, the script will use cosmosDBIdentityClientId to get the access token.
+    - cosmosDBServicePrincipalTenantId cannot be null when cosmosDBServicePrincipalClientSecret is provided.
+    - Both targetVMSubRGNameList (list of target VMs) and targetVMSSSubRGName (target VMSS) cannot be null at the same time. At least one target is needed. Both can be specified together.
+    - To target a VMSS for fault, VMSSInstanceIdList should specify which VM instances in the VMSS need to be targeted e.g. {0,1,2}. 
+
+### Usage
+
+To use the Cosmos DB client-side network chaos tool, follow these steps:
+
+1. On the **Client VM** mentioned in the Prerequisites section, open a PowerShell terminal with **Admin Privileges** and navigate to the directory where you saved the tool.
+2. You may want to Run the following command to bypass powershell checks:
+``` 
+Powershell -ExecutionPolicy Bypass
+```
+3. For creating ```Network Outage Chaos```
+
+    - Execute the tool by running the following command making sure to replace all the placeholders with appropriate values:
+    ```
+    .\cosmosdb_client_network_chaos_tool.ps1 `
+    -cosmosDBEndpoint "<cosmosDBEndpointUrl>" `
+    -databaseId "<databaseId>" `
+    -containerId "<containerId>" `
+    -faultRegion "<faultRegion>" `
+    -chaosStudioSubscriptionId "<chaosStudioSubscriptionId>" `
+    -chaosStudioResourceGroupName "<chaosStudioResourceGroupName>" `
+    -chaosStudioManagedIdentityClientId "<chaosStudioManagedIdentityClientId>" `
+    -chaosExperimentName "<chaosExperimentName>" `
+    -chaosExperimentManagedIdentityName "<chaosExperimentManagedIdentityName>" `
+    -durationOfFaultInMinutes <durationOfFaultInMinutes> `
+    -targetVMSubRGNameList "<targetVMSubRGNameList>" `
+    -targetVMSSSubRGName "<targetVMSSSubRGName>" `
+    -vmssInstanceIdList "<vmssInstanceIdList>"
+    ```
+
+    - Following is an example to understand the input formatting:
+    ```
+    .\cosmosdb_client_network_chaos_tool.ps1 -cosmosDBEndpoint "https://mycosmosdb.documents.azure.com:443/" -databaseId "mydatabase" -containerId "mycontainer" -faultRegion "East US" -chaosStudioSubscriptionId "12345678-1234-1234-1234-1234567890ab" -chaosStudioResourceGroupName "chaos-rg" -chaosStudioManagedIdentityClientId "87654321-4321-4321-4321-210987654321" -chaosExperimentName "myexperiment" -chaosExperimentManagedIdentityName "experiment-mi" -durationOfFaultInMinutes 10 -targetVMSubRGNameList "{12345678-1234-1234-1234-1234567890ab/rg1/vm1,12567841-4321-4321-1234-1234567890gh/rg2/vm2}" -targetVMSSSubRGName "12345678-1234-1234-1234-1234567890ab/rg1/vmss" -vmssInstanceIdList "{0,1,2}"
```

Comment:
nit: apply new line format to all commands in the so it's consistent. 
```suggestion
    .\cosmosdb_client_network_chaos_tool.ps1 `
    -cosmosDBEndpoint "https://mycosmosdb.documents.azure.com:443/" `
    -databaseId "mydatabase" `
    -containerId "mycontainer" `
    -faultRegion "East US" `
    -chaosStudioSubscriptionId "12345678-1234-1234-1234-1234567890ab" `
    -chaosStudioResourceGroupName "chaos-rg" `
    -chaosStudioManagedIdentityClientId "87654321-4321-4321-4321-210987654321" `
    -chaosExperimentName "myexperiment" `
    -chaosExperimentManagedIdentityName "experiment-mi" `
    -durationOfFaultInMinutes 10 `
    -targetVMSubRGNameList "{12345678-1234-1234-1234-1234567890ab/rg1/vm1,12567841-4321-4321-1234-1234567890gh/rg2/vm2}" `
    -targetVMSSSubRGName "12345678-1234-1234-1234-1234567890ab/rg1/vmss" `
    -vmssInstanceIdList "{0,1,2}"
```

Reply:
Done